Makefile

```diff
@@ -35,21 +35,15 @@ BUCKET=rightscale-binaries
 ACL=public-read
 # version for gopkg.in, e.g. v1, v2, ...
 GOPKG_VERS=v8
-GLIDE_VERSION?=v0.13.3
 GOOS=$(shell go env GOOS)
 GOARCH=$(shell go env GOARCH)
-ifeq (windows,$(GOOS))
-GLIDE_EXEC=glide-$(GLIDE_VERSION).exe
-else
-GLIDE_EXEC=glide-$(GLIDE_VERSION)
-endif
-# Dependencies not handled by glide, i.e. that are used to build/test/upload this puppy
+# Dependencies used to build/test/upload this puppy
 DEPEND=golang.org/x/tools/cmd/cover \
 			github.com/onsi/ginkgo \
 			github.com/onsi/ginkgo/ginkgo \
 			github.com/golang/protobuf/proto \
 			github.com/rlmcpherson/s3gof3r/gof3r \
-			github.com/rogpeppe/govers
+			github.com/tylercasper/govers
```

Comment:
Fork this dependency under Rightscale org

Reply:
😕 Do we really need `govers` or even `gopkg.in` still? I think we can probably move to a release strategy that does not use them and uses tags instead.

Reply:
@douglaswth Probably true. I think we should create a separate ticket for this so that the pending rsc-related changes can be released.

Reply:
> Fork this dependency under Rightscale org

I need additional permissions to fork under Rightscale org and Flexera org